vignettes/epinowcast.Rmd

```diff
@@ -26,7 +26,7 @@ In this quick start, we demonstrate using `epinowcast` to specify and fit a mini
 
 # Package
 
-In this quick start, we also use `data.table` and `ggplot2` packages. These are both installed as dependencies when `epinowcast` is installed. Note that all output from `epinowcast` is readily useable with  other tools, including `tidyverse` packages (see [here](https://mgimond.github.io/rug_2019_12/Index.html) for a comparison).
```

Comment:
Ah damn. Can you revert this and I will fix by adding these changes to the non-rendered version? It looks like the recent PR (#406) accidently targeted the wrong thing.

Reply:
Alright. See fc18c2680d3645822c8aa5adefa5748864ae3ca1.